packages/cli/src/metro-config.ts

```diff
@@ -130,6 +131,13 @@ export function customizeMetroConfig(
   const metroConfig = metroConfigReadonly as InputConfigT;
 
   const plugins: MetroPlugin[] = [];
+  if (customSerializationPlugins) {
```

Comment:
We should ensure we have an array here:
```suggestion
  if (Array.isArray(customSerializationPlugins)) {
```

Reply:
Btw, you don't need the extra if check, `Array.isArray` accepts `null` and `undefined`.